### ISSUE TITLE
Extension not working for git repositories that contain a . character.

### ISSUE THREAD

**Issue by mark-truran** [thieman/github-selfies#88]
Example url: https://www.github.com/exampleowner/example.repository/pulls/1

Github selfie doesn't like the . in example.repository and therefore fails to load into the page.

Potentially the 'allowed paths' regex list? @ https://github.com/thieman/github-selfies/blob/master/chrome/selfie.js#L37

I can't selfie my company repositories because of this. Please fix.

**Comment by ChaseWagoner** [comment]
When attempting to sign up for GitHub:
>Username may only contain alphanumeric characters or single hyphens, and cannot begin or end with a hyphen

When attempting to create a repository, GitHub allows alphanumeric, periods, hyphens, and underscores; anything else is converted to a hyphen.

However, these rules are more strict than the `allowedPaths` need to be; since usernames and repository names, by rule, can't contain forward slashes, the pattern could be changed to accept any character until the next forward slash, for example:
```javascript
// Any non-slash character
/github.com\/[^\/]+\/[^\/]+\/issues\/new/
// Or a lazy wildcard
/github.com\/.+?\/.+?\/issues\/new/
```

**Comment by thieman** [comment]
This was hopefully fixed by #94 which rolled out in version 2.0.5. Going to close this issue, if it's still a problem please reopen.
